docs/user/assessment.rst

```diff
@@ -0,0 +1,167 @@
+            :align: center
+
+
+Integrated Training Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Instructors can use the assessment training process to make the grading more consistent. An integrated training process for reviewers based on example submissions and example assessments ensures that reviewers have enough knowledge to properly assess submissions and provide feedback. They define a series of example submissions and assessments that the tutors must first read through.
+
+Double-blind Grading
+^^^^^^^^^^^^^^^^^^^^
+
+The manual assessment begins after the deadline of an exercise has passed for all students and is **double-blind**. It means that the tutors do not know the names of the students they assess, and the students do not know the identity of the tutors. The double-blind grading aims to minimize bias and increase the objectivity of the assessment. It implies that both the students and the tutors are "blind" to the identity of each other, ensuring that their expectations or biases do not influence the results. 
+    
+    .. figure:: assessment/double-blind.png
+            :alt: Double-blind grading
+            :align: center
+   
+.. _exercise_complaints:
+
+Complaints
+^^^^^^^^^^
+
+After receiving a grade, students can complain about an exercise assessment if the instructor enabled this option, the complaint deadline is still ongoing, and the students think the evaluation needs to be revised. 
+The instructor can set a maximum number of allowed complaints per course. These so-called tokens are used for each complaint. The token is returned to the student if the tutor accepts the complaint. 
+It means a student can submit as many complaints as they want, as long as they are accepted.
+
+The complaint process is as follows:
+
+1. The student opens the related exercise, interacts with the "Complain" button below the exercise instructions, and writes additional text before submitting a complaint to justify the reevaluation.
+   
+   .. figure:: assessment/complaint-submission.png
+            :alt: Complaint submission
+            :align: center
+
+2. The reviewer interacts with the "Assessment Dashboard" button of the desired course, which displays the table for all the course exercises.
+
+    +---------------------------------------------------------+--------------------------------------------------+
+    | .. figure:: assessment/assessment-dashboard-button.png  | .. figure:: assessment/assessment-dashboard.png  |
+    |    :alt: Assessment Dashboard button                    |    :alt: Assessment Dashboard                    |
+    +---------------------------------------------------------+--------------------------------------------------+
+
+
+3. By interacting with the respective "Exercise Dashboard" button, the reviewer opens the exercise-specific dashboard and assess students' submissions. 
+   Upon evaluation, the reviewer puts a "lock" on the complaint disabling inconsistency and ambiguity, which expires in 24 hours.
```

Comment:
This is correct but locks are also put when there's no complaint. If the tutor starts assessing any submission (without complaint) it gets locked and as far as I know the lock is not released until the tutor cancels or submits the assessment. I think this information should also be mentioned somewhere.